src/pages/learn.js

```diff
@@ -290,16 +308,18 @@ const LearnPage = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
         [filterActiveArticles, pathname]
     );
+
     useEffect(() => {
         updateFilterQueryParams(filterValue);
     }, [filterValue, updateFilterQueryParams]);
+
     // filterValue could be {} on a page load, or values can be "all" if toggled back
-    const hasNoFilter = useMemo(
-        () =>
+    const hasNoFilter = useMemo(() => {
+        return (
             (!filterValue.languages || filterValue.languages === 'all') &&
-            (!filterValue.products || filterValue.products === 'all'),
-        [filterValue]
-    );
+            (!filterValue.products || filterValue.products === 'all')
+        );
+    }, [filterValue]);
```

Comment:
This is the same functionally as before right

Reply:
It is. I added the curly brackets so that I could console log in the function. I wanted to know how useMemo works